Source/Initialization/WarpXInitData.cpp

```diff
@@ -306,3 +474,116 @@ WarpX::InitLevelDataFFT (int lev, Real time)
 }
 
 #endif
+
+/* \brief
```

Comment:
```
/**
 * \brief
```
And could you please put it in the .H file?

Reply:
Done. Thank you